os/kernel/init/os_start.c

```diff
@@ -488,7 +488,11 @@ void os_start(void)
 		 * the IDLE task.
```

Comment:
Can this comment be moved into else case of below as it doesnt apply in AMP case?

```
               /* Set the affinity mask to allow the thread to run on all CPUs.  No,
		 * this IDLE thread can only run on its assigned CPU.  That is
		 * enforced by the TCB_FLAG_CPU_LOCKED which overrides the affinity
		 * mask.  This is essential because all tasks inherit the affinity
		 * mask from their parent and, ultimately, the parent of all tasks is
		 * the IDLE task.
		 * /
```

Reply:
Done